VocaDbWeb/tsconfig.json

```diff
@@ -25,7 +25,8 @@
 		},
 		"resolveJsonModule": true,
 		"experimentalDecorators": true,
-		"emitDecoratorMetadata": true
+		"emitDecoratorMetadata": true,
+		"useDefineForClassFields": true

```

Comment:
See https://github.com/mobxjs/mobx/issues/2843#issuecomment-794137196 and [Migrating from MobX 4/5 · MobX](https://mobx.js.org/migrating-from-4-or-5.html).
> Add the flag `"useDefineForClassFields": true` to your compiler config.